.claude/skills/docs-consistency-checker/details/validation-rules.md

```
@@ -0,0 +1,81 @@
+**검증**: MDX 컴포넌트의 ID가 Rootage metadata.id와 일치
+
+```markdown
+<PlatformStatusTable componentId="action-button" />  # Must match metadata.id
+<ComponentSpecBlock id="action-button" />             # Must match metadata.id
+```
+
+## 5. File Existence Check
+
+**검증**: Rootage YAML 존재 시 관련 문서도 존재해야 함
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)